src/model.py

```diff
@@ -1912,6 +1919,24 @@ def get_confidence(expression_logits, batch, current_epoch):
         return expression_confidence  # shape: [batch, sequence]
 
 
+    def transmission(self, logits, true):
+        """
+        To help guide the attention mechanisms on which tokens to focus more on,
+        given logits from individual subtask. 
+        """
+        # decide how much true labels should influence attention based on current epoch
+        gold_influence = self.get_prob(self.current_epoch, self.find("warm_up_constant", default=5))
+
+        focus_scope = (  # strengthen signal from true logits
+            gold_influence*true.bool().float()
+                + (1-gold_influence)*logits.argmax(-1)
+        ).detach().to(torch.device(self.device))
+        focus_scope.requires_grad = True
+        
+        return focus_scope  # shape: [batch, sequence]
```

Comment:
Copied directly from FgFlex implementation. 

Should also update FgFlex to use logits, not cnn outputs, since logits will the entire subtask's information.